frontend/packages/kubevirt-plugin/integration-tests/tests/cdi-upload.scenario.ts

```diff
@@ -0,0 +1,92 @@
+import { execSync } from 'child_process';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { errorMessage, saveChangesBtn } from '@console/internal-integration-tests/views/crud.view';
+import { pvcStatus } from '@console/ceph-storage-plugin/integration-tests/views/pvc.view';
+import { PVC_STATUS } from '@console/ceph-storage-plugin/integration-tests/utils/consts';
+import { CDI_UPLOAD_TIMEOUT_SECS, STORAGE_CLASS } from './utils/constants/common';
+import { OperatingSystem } from './utils/constants/wizard';
+import * as cdiUploadView from '../views/cdiUploadView';
+import { PVCData } from './types/pvc';
+import { UploadForm } from './models/pvcUploadForm';
+
+// this scenario tests the upload procedure, without creating a vm from the uploaded resource.
+// https://issues.redhat.com/browse/CNV-6020
+
+describe('KubeVirt CDI Upload', () => {
+  const uploadForm = new UploadForm();
+  const srcImage =
+    'http://cnv-qe-server.rhevdev.lab.eng.rdu2.redhat.com/files/cnv-tests/cirros-images/cirros-0.4.0-x86_64-disk.qcow2';
+  const desImage = '/tmp/cirros.qcow2';
+  const invalidImage = '/tmp/cirros.txt';
+  const pvcName = `upload-pvc-${testName}`;
+
+  beforeAll(async () => {
+    execSync(`curl ${srcImage} -o ${desImage}`);
+    execSync(`ln -s -f ${desImage} ${invalidImage}`);
+  });
+
+  afterAll(async () => {
+    execSync(`kubectl delete -n ${testName} dv ${pvcName}`);
```

Comment:
shouldn't we also delete the downloaded image?

Reply:
Updated, delete the downloaded image also.